stmt.go

```diff
@@ -639,6 +640,14 @@ func (s *Stmt) Columns(dest []any) error {
 	}
 
 	types := util.View(s.c.mod, typePtr, count)
+
+	// hint to the compiler that
+	// it can omit bounds check
+	// accessing types[i] below.
+	if len(types) != len(dest) {
+		return errors.New("incorrect destination count")
```

Comment:
If this can never happen (I don't think it can):
```go
panic(util.AssertErr())
```

This increases the usefulness of code coverage for me (as my tooling ignores these lines).